.circleci/config.yml

```diff
@@ -424,6 +430,9 @@ jobs:
       TEST_NAME: hello-world
     steps:
       - checkout
+      - run:
+          name: wait for starter
+          command: docker exec -it cfy_manager cfy_manager wait-for-starter
```

Comment:
The wait-for-starter is available only on 5.1.0 not on 505 so you do not need to have it here